examples/testdata/path_matcher/openapi_swagger.json

```diff
@@ -72,5 +72,11 @@
     "https"
   ],
   "swagger": "2.0",
-  "x-google-allow": "all"
+  "x-google-allow": "all",
+  "x-google-endpoints": [
+    {
+      "allowCors": true,
```

Comment:
I can see: "allowCors" is supported, but "allow: all" is not supported. 

In the Envoy route match,  we have a route match for "option" for /libraries/{library_id}/shelves/{shelf_id}
we don't have any route match for /** for any methods.

Reply:
Yup, we OpenAPI compiler doesn't generate backend rules for `x-google-allow: all`. So it's not supported for dynamic routing.